modules/openstack/user.go

```diff
@@ -71,6 +71,9 @@ func (o *OpenStack) CreateUser(
 	return userID, nil
 }
 
+// UserNotFound - user not found error message"
+const UserNotFound = "user not found in keystone"
+
```

Comment:
Same placement question here as for the new service const

Reply:
done